content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -539,6 +539,19 @@ Used on: All Objects
 
 This annotation is used for describing specific behaviour of given object.
 
+### kubernetes.io/display-name {#display-name}
+
+Type: Annotation
+
+Example: `kubernetes.io/display-name: "Display name of K8s object."`
```

Comment:
```suggestion
Example: `kubernetes.io/display-name: "Displays the name of a K8s object."`
```

Reply:
To me `"Display name of K8s object."` is the more accurate example. After all, it is the _display name_.

Reply:
Sure, I was just copying the above example. This feels like a preference, but I'll go with it.